bundles/http/Citrix/Cherwell/Cherwell.js

```diff
@@ -0,0 +1,1278 @@
+                        "name": "Service",
+                        "type": "STRING",
+                        "length": 32
+                    },
+                    {
+                        "name": "IncidentType",
+                        "type": "STRING",
+                        "length": 32
+                    }
+                ]
+            },
+            {
+                "name": "servicesIncidentTypes",
+                "columns": [
+                    {
+                        "name": "id",
+                        "type": "LONG",
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "Service",
+                        "type": "STRING",
+                        "length": 32
+                    },
+                    {
+                        "name": "IncidentType",
+                        "type": "STRING",
+                        "length": 32
+                    }
+                ]
+            },
+            {
+                "name": "categoriesIncidentTypes",
+                "columns": [
+                    {
+                        "name": "id",
+                        "type": "LONG",
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "Category",
+                        "type": "STRING",
+                        "length": 64
+                    },
+                    {
+                        "name": "IncidentType",
+                        "type": "STRING",
+                        "length": 64
+                    }
+                ]
+            },
+            {
+                "name": "sources",
+                "columns": [
+                    {
+                        "name": "RecID",
+                        "type": "STRING",
+                        "length": 45,
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "Source",
+                        "type": "STRING",
+                        "length": 64
+                    }
+                ]
+            },
+            {
+                "name": "priorities",
+                "columns": [
+                    {
+                        "name": "RecID",
+                        "type": "STRING",
+                        "length": 45,
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "Priority",
+                        "type": "INTEGER"
+                    }
+                ]
+            }
+        ]
+    }
+});
+
+const pageSize = 200;
+
+async function getRequest({ dataStore, client }, moduleName, body, endpoint) {
+    let requestOptions = {
+        method: 'POST',
+        body: JSON.stringify(body)
+    };
+
+    let resp = await client.fetch(endpoint, requestOptions);
+    let moreRecords = true;
+    if (resp.ok && resp.status === 200) {
```

Comment:
`resp.satatus === 200` is redundant, it's already included in `resp.ok`

Reply:
In this case the 200 it's not redundant, Cherwell api also has 201 and 202 responses, and we specifically need the 200 status, ex 201 means response its ok but no more records are found